msl/equipment/resources/isotech/millik.py

```diff
@@ -133,11 +133,11 @@ def read_channel(self, channel: int, n: int = 1) -> float | list[float]:
 
         return readings
 
-    def read_all_channels(self, n: int = 1) -> list[float]:
+    def read_all_channels(self, n: int = 1) -> tuple[list[float], list[float]]:
```

Comment:
I believe that the channel type is an `int`, so the return type should be `tuple[list[int], list[float]]`